http/bench/dune

```diff
@@ -1,3 +1,9 @@
 (executable
  (name bench)
  (libraries http core core_bench))
+
+(rule
+ (alias runtest)
+ (package http)
+ (action
+  (run ./bench.exe)))
```

Comment:
I don't think this should be done. The benchmarks are quite slow.

Reply:
It is just 10 extra seconds, we could reduce it further to 1 or 2
seconds or remove it. If we leave it there the ocaml-CI fails. 

An alternative would be to delete the dune file and put there a README for building and running the benchmarks

Reply:
Why does it fail if we leave it?

Reply:
`dune build` in the GH CI tries to build everything, as well as `dune build @all` in ocaml CI. But I was thinking to also remove core_bench from the test dependencies. You would leave it there?

Reply:
Maybe you could create a separate `cotthp-benchmarks` package that correspond to the benchmarks (not installed by default), and includes the benchmarking dependencies?

Reply:
Sounds nice, it would also be a place to put test servers to use for benchmarking purposes

Reply:
>  But I was thinking to also remove core_bench from the test dependencies. You would leave it there?

Yes, I would leave it there. You can also make sure the tests build the benchmark suite but not run it.

Gasche's suggestion also sounds fine.

Reply:
I am going to leave it here for now. And open a separate issue to discuss the opportunity to have a package with executables for testing